test/mocks/ERC20TransferTax.sol

```diff
@@ -2,27 +2,16 @@
 
 pragma solidity 0.8.10;
 
-import "openzeppelin/token/ERC20/ERC20.sol";
+import {ERC20Mock} from "./ERC20.sol";
 
-/// @title ERC20MockDecimals
+/// @title ERC20MockTransferTax
 /// @author Trader Joe
 /// @dev ONLY FOR TESTS
-contract ERC20WithTransferTax is ERC20 {
+contract ERC20TransferTaxMock is ERC20Mock {
     /// @dev Constructor
-    constructor() ERC20("ERC20Mock", "ERC20M") {}
+    constructor() ERC20Mock(18) {}
 
-    /// @dev Mint _amount to _to.
-    /// @param _to The address that will receive the mint
-    /// @param _amount The amount to be minted
-    function mint(address _to, uint256 _amount) external {
-        _mint(_to, _amount);
-    }
-
-    function transferFrom(
-        address from,
-        address to,
-        uint256 amount
-    ) public override returns (bool) {
+    function transferFrom(address from, address to, uint256 amount) public override returns (bool) {
```

Comment:
Instead of overwriting both transfer function, you can overwrite `_transfer` directly